docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+
+  - id: appsfs-create
+    source: modules/file-system/filestore
+    use: [network-create]
+    settings: {name: apps}
+
+  - id: bucket-software-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_software)
+
+  - id: bucket-input-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_inputs)
+
+  - id: bucket-output-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_outputs)
+
+- group: software_installation
+  modules:
+
+  - id: network-install
+    source: modules/network/pre-existing-vpc
+
+  - id: appsfs-install
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /apps
+      fs_type: nfs
+
+  ### Software ###
+
+  - id: spack
+    source: community/modules/scripts/spack-install
+    settings:
+      install_dir: /apps/spack
+      configs:
+      - type: file
+        scope: defaults
+        content: |
+          config:
+            build_stage:
+              - /apps/spack/spack-stage
+      - type: file
+        scope: defaults
+        content: |
+          modules:
+            default:
+              tcl:
+                hash_length: 0
+                all:
+                  conflict:
+                    - '{name}'
+                projections:
+                  all: '{name}/{version}-{compiler.name}-{compiler.version}'
+      - type: file
+        scope: site
+        content: |
+          packages:
+            slurm:
+              externals:
+                - spec: slurm@21-08-8-2
+                  prefix: /usr/local
+              buildable: False
+      compilers: [gcc@9.3.0 target=x86_64]
+      environments:
+      - name: gromacs
+        type: file
+        content: |
+          spack:
+            definitions:
+            - compilers:
+              - gcc@9.3.0
+            - mpis:
+              - intel-mpi@2018.4.274
+            - mpi_packages:
+              - gromacs@2020.5
+            specs:
+            - matrix:
+              - - $mpis
+              - - $%compilers
+            - matrix:
+              - - $mpi_packages
+              - - $%compilers
+              - - $^mpis
+      # Un-comment and update mirror_url to install from spack cache
+      # spack_cache_url:
+      # - mirror_name: gcs_cache
+      #   mirror_url: gs://...
+
+  - id: spack-builder-startup  # TODO: Update example or make generic
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(spack.install_spack_deps_runner)
+      - $(spack.install_spack_runner)
+      - type: shell
+        destination: setup_gromacs.sh
+        content: |
+          #!/bin/bash
+          source /apps/spack/share/spack/setup-env.sh
+          spack env activate gromacs
```

Comment:
I am unsure what is the impact of having lines 176 and 177. It seems that setup-env.sh and spack env activate just load the environment and have no long lasting effect on the next commands.  
Since you are not actually running gromacs here, are these lines needed?

Reply:
There were a bad artifact. They have been deleted.